robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -44,26 +68,50 @@ public Limelight(NetworkTable table) {
         snapshot = table.getEntry("snapshot");
     }
 
+    /*
+     * Returns true if the Limelight has a target, else false
+     * @returns A boolean value stating whether the Limelight has a target
+     */
     public boolean hasTarget() {
         return MathUtils.epsilonEquals(tv.getDouble(0), 1);
     }
 
+    /*
+     * Returns the area of the target divided by the size of the image
+     * @returns A value from 0.0 to 1.0 representing the target area
+     */
     public double getTargetArea() {
         return ta.getDouble(0);
     }
 
+    /*
+     * Returns the position of the target in degrees within the image
+     * @returns A Vector2 representing the position of the target
```

Comment:
We know its a vector2, just say it returns the position of the target.
Also the position is in radians not degrees

Reply:
odne